### ISSUE TITLE
Issue with Chart Dimensions Not Updating in position_json After Resizing Dashboard Column

### ISSUE THREAD

**Issue by lauslzrm** [apache/superset#31245]
Hello!

I've been experiencing an issue lately that I'm not sure where it originates from. Currently, I'm using **version 4.0.2.** When I create a dashboard and add a column, for some reason, when I add a chart to that column and later enlarge the column, the new dimensions of my chart are not updated correctly in the position_json configuration. As a result, when using the API to download the chart's image, it downloads with incorrect dimensions. I've attached images to make it a bit clearer:

Image 1: I create a column in my dashboard. 
![image1](https://github.com/user-attachments/assets/fb80bcc5-98c7-4829-8d95-5df70c0f2c57)

Image 2: I add a chart to my column.
![image2](https://github.com/user-attachments/assets/c8259757-9354-49f9-ae96-3007d61f8280)

Image 3: I enlarge the size of my column.
![image3](https://github.com/user-attachments/assets/f38e111e-3708-4b34-95c7-75bc44450048)

Image 4: When I remove my chart from the column, it doesn't maintain the resized dimensions after enlarging the column.
![image](https://github.com/user-attachments/assets/97a03f5f-0b34-4a19-8f79-63b006f44e05)

In summary, and just to clarify, my problem is that when varying the dimensions of a column, these changes are not reflected in the position_json that I access via the API to download the dashboard's images. Furthermore, you don't necessarily need to look at the JSON to observe the issue, as it can also be seen directly in the dashboard: when extracting the chart from the column, it reverts to its default or original size as when a chart is first added to the dashboard.

Currently, as a workaround, I adjust my charts to the desired size before adding them to a column, which resolves the issue. However, I believe it's important to highlight this in case it's behavior that should not be happening.

Thank you

**Comment by rusackas** [comment]
Can you validate that this is still happening in 4.1.2 / 5.0.0rc2?

**Comment by rusackas** [comment]
We haven't heard back from anyone that this is still happening in current releases (or on `master`) and I haven't encountered it or seen any other reports. We'll close this out, but if you can validate the bug is still currently happening, we can happily reopen/revisit.
